pkg/controller/vitesskeyspace/vitesskeyspace_controller.go

```diff
@@ -179,6 +184,23 @@ func (r *ReconcileVitessKeyspace) Reconcile(request reconcile.Request) (reconcil
 	topoResult, err := r.reconcileTopology(ctx, vtk)
 	resultBuilder.Merge(topoResult, err)
 
+	wr, err := newWrangler(ctx, *vtk)
+	if err != nil {
+		r.recorder.Eventf(vtk, corev1.EventTypeWarning, "StatusUpdateWarning", "failed to retrieve resharding information: %v", err)
+	}
+	if wr != nil {
```

Comment:
Should we hit this? I think we should guarantee that if err == nil that wr will be valid.

Reply:
Was an artifact of a previous version of the code when it lived in a different file, it's gone now!